src/components/Editor/Preview/Popup.js

```diff
@@ -46,24 +53,32 @@ export class Popup extends Component<Props> {
   marker: any;
   pos: any;
 
-  componentDidMount() {
+  async componentWillMount() {
     const {
-      loadObjectProperties,
-      loadedObjects,
       value,
-      editor,
-      range
+      expression,
+      loadObjectProperties,
+      loadedObjects
     } = this.props;
+    const root = createNode(null, expression, expression, { value });
+
+    if (!nodeIsPrimitive(root) && value.actor && !loadedObjects[value.actor]) {
```

Comment:
before, the value was potentially null -- is that no longer the case? maybe we can type this?